include/clang/AST/ASTStructuralEquivalence.h

```diff
@@ -74,10 +72,10 @@ struct StructuralEquivalenceContext {
 
   /// Determine whether the two declarations are structurally
   /// equivalent.
-  bool IsStructurallyEquivalent(Decl *D1, Decl *D2);
+  bool IsEquivalent(Decl *D1, Decl *D2);
```

Comment:
Note I renamed the toplevel interface functions to `IsEquivalent` so we can easily distinguish them from the static implementation functions.